Update two readme files for public preview pnp (#1549)
Azure/azure-iot-sdk-c: digitaltwin_client/samples/readme.md

## Directory contents

### Executable Source Code

> * [digitaltwin\_sample\_device](./digitaltwin_sample_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using Iot Hub.

Reviewer: Maybe @jspaith can confirm but I think the "T" in IoT is [capitalized](https://azure.microsoft.com/en-us/services/iot-hub/)
```suggestion
* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - This executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation when using IoT Hub.
```

> * [digitaltwin\_sample\_ll_device](./digitaltwin_sample_ll_device) - The executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation interfaces and basic operation.  It uses the lower level (\_LL\_) layer and enables the Device Provisioning Service (DPS) to connect to Iot Central. It is appropriate for devices with limited resources.

Reviewer: ```suggestion
* [digitaltwin\_sample\_ll_device](./digitaltwin_sample_ll_device) - This executable interacts with the Digital Twin interfaces of the neighboring libraries, demonstrating creation of interfaces and basic operation.  It uses the lower level (\_LL\_) layer and enables the Device Provisioning Service (DPS) to connect to IoT Central. It is appropriate for devices with limited resources.
```

### Libraries
* [digitaltwin\_sample\_device_info](./digitaltwin_sample_device_info) - Library that implements a sample device info Digital Twin interface of helper functions.  This interface reports information about the device - such as OS version, amount of storage, etc.
* [digitaltwin\_sample\_environmental_sensor](./digitaltwin_sample_environmental_sensor) - Library that implements a sample environmental sensor Digital Twin interface of helper functions.  This interface demonstrates all concepts of implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.

## Getting started
Build the Digital Twin Client SDK, if not already completed.  Follow instructions [here](../doc/building_sdk.md).

### digitaltwin\_sample\_device

* Use cmake to build the [digitaltwin\_sample\_device](digitaltwin_sample_device) directory.

  From the azure-iot-c-sdk repository root:
  ```

>   cd cmake

Reviewer: This assumes they have already done the
```bash
cmake -D<whatever-the-option-is-for-digital-twin> ..
```
correct?

Author: Correct.  That step is covered in the "Build the Digital Twin Client SDK" instruction link on line 16.  (/digitaltwin_client/doc/building_sdk.md)

  cmake --build .
  ```
  This will build the `digitaltwin_sample_device` executable along with the aforementioned sample interfaces it uses.

> * Get the Connection String of the Iot Hub device you wish to DigitalTwin enable.  

Reviewer: ```suggestion
* Get the Connection String of the IoT Hub device you wish to enable for Digital Twin.  
```

  This can be found via the [azure portal](https://portal.azure.com):  
    1. Go to your IoT Hub
    2. Under Explorers in the left pane, select IoT Devices
    3. Under Device ID, select a device.  Else, add a new device and then select it.
    4. Copy the Primary Connection String

* Run the sample.

  **If using Windows:** 

  From the cmake directory:
  ```
  cd .\digitaltwin_client\samples\digitaltwin_sample_device\Debug\
  ```
  Run the executable using your connection string, e.g.:
  ```
  .\digitaltwin_sample_device.exe "HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret"
  ```

>   **If using Linux:**

Author: I wonder if it warrants specifying here which version of Linux is supported.  I tested in Ubuntu 18.04.  The "Compiling the C Device SDK" instructions for Linux on a prior connected page is only for Ubuntu.  If so, should it also specify which version of Windows is supported?

* [digitaltwin\_sample\_environmental_sensor](./digitaltwin_sample_environmental_sensor) - This directory builds the samples environmental sensor library.  This is a Digital Twin interface of helper functions that demonstrates all concepts of implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.

## Getting started

* Build the Digital Twin Client SDK, if not already completed. See [here](../doc/building_sdk.md)

### digitaltwin\_sample\_device

* Use CMake to build the [digitaltwin\_sample\_device](digitaltwin_sample_device) directory.

  From the cmake directory:
  ```
  cmake --build .
  ```
  This will build the `digitaltwin_sample_device` executable along with the aforementioned sample interfaces.

* Copy the Connection String of the Azure IoT Hub device you wish to enable for Digital Twin:

* Run the sample.

  From the cmake directory:
  
  **If using Windows:** 

  ```
  cd .\digitaltwin_client\samples\digitaltwin_sample_device\Debug\
  ```

>   Run the executable using your connection string, e.g.:

Reviewer 2: Nit: I think you can get rid of the "run the executable" live here and a few lines below.  In the sample_ll_device below you have both the cd and execution step inside the same ``` block and I think it's easier to read like that.  Users just need to copy/paste, making appropriate changes if needed, and don't need to read interspersed text with additional commentary in this case.

  ```

  **If using Linux:**

  ```
  cd digitaltwin_client/samples/digitaltwin_sample_device
  ```
  Run the executable using your connection string, e.g.:
  ```
  ./digitaltwin_sample_device "HostName=yourHub;DeviceId=yourDigitalTwinDeviceId;SharedAccessKey=secret"
  ```

  Note: In both OS options, be sure to include quotations around the Connection String.

### digitaltwin\_sample\_ll_device

* Configure the files.

  Please go [here](./digitaltwin_sample_ll_device) to follow the configuration steps before building the SDK.

* Use cmake to build the [digitaltwin\_sample\_ll_device](digitaltwin_sample_ll_device) directory.

  ```
  cmake --build .
  ```
  This will build the `digitaltwin_sample_ll_device` executable along with the aforementioned sample interfaces.
  
* Run the sample.

  From the cmake directory:

  **If using Windows:** 
  ```
  cd .\digitaltwin_client\samples\digitaltwin_sample_ll_device\Debug\

  .\digitaltwin_sample_ll_device.exe ..\..\..\..\..\digitaltwin_client\samples\digitaltwin_sample_ll_device\sample_config\dpsSymmKey.json
  ```

  **If using Linux:**
  ```
  cd digitaltwin_client/samples/digitaltwin_sample_ll_device/
  

>   ./digitaltwin_sample_ll_device dpsSymmKey.json ../../../../../digitaltwin_client/samples/digitaltwin_sample_ll_device/sample_config/dpsSymmKey.json

Reviewer 2: "./digitaltwin_sample_ll_device dpsSymmKey.json .." - remove the first dpsSymmKey.json that comes immediately after digitaltwin_sample_ll_device, since the real source of this is the ..
